### ISSUE TITLE
Spinner instead of "Starting...."

### ISSUE THREAD

**Issue by marcelogaio** [tingbot/tbprocessd#6]
It'd be nice to have a spinner (animated gif) instead of the "starting /apps/..." message.
The message itself is often truncated, so it's not very useful.
Keep up the good work. Tingbot rocks!
Cheers!

**Comment by joerick** [comment]
There are actually two screens visible when tapping an app from springboard - the first is rendered by the Springboard, and the second is a console message displayed by tbprocessd.

We could improve the first with an animated GIF, though I'm unsure if it would continue animating while the springboard is closing. The second is just a static console message, so there's not really much potential to improve that one!

I do think this flow could use some improvement. The repeated messages aren't very useful. Maybe tbprocessd could do some ANSI background colored output to make it a bit more exciting?!

**Comment by marcelogaio** [comment]
Yeah, I saw the 2 steps you mention. I was unsure about the tbprocessed handling graphincs.
The first part as you say could have a spinner but the animation part is tricky...

Regarding the tbprocessd part, I was about to ask about the graphics capability, but if it's a console, maybe the colored output... or a typical linux-text-loading-spinner "##########"...
Anyway, just a thought!

Cheers!

**Comment by joerick** [comment]
The tbprocessd output will have to be static, because tbprocessd has already started the next app, and it can output to the console (e.g. if it’s building a virtualenv).

I do like the idea of using coloured background… hmmm…

Thanks!
